app/workers/organizations/bust_cache_worker.rb

```diff
@@ -0,0 +1,15 @@
+module Organizations
+  class BustCacheWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority, retry: 10
+
+    def perform(organization_id, slug, cache_buster = CacheBuster)
```

Comment:
Since we are not using this `CacheBuster` argument option, could you remove it and just explicitly use `CacheBuster` below? This will simplify the worker a bit and avoid unnecessary complexity

Reply:
Hi @mstruve, thanks for your review! I will do the changes you asked, also fix the failing spec :)